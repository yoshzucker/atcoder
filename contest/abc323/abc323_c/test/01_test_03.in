100 100
2000 2100 2100 1300 1300 1800 1800 800 800 1700 2400 2400 1900 1900 2500 800 1600 1600 1600 1900 2500 2100 1800 2100 700 1000 1600 1100 600 1300 1300 1600 1700 900 1000 2300 600 1900 2000 2000 1200 2100 1700 600 2000 1900 2300 600 1100 2500 1800 1700 1000 900 1300 1100 2100 2200 2100 900 500 1700 2400 1900 800 2000 1800 1100 1300 2200 1100 2300 800 900 1200 600 700 1000 1300 2300 2200 1500 1800 1700 1900 2100 1200 500 1200 1300 1600 1300 900 2200 600 1500 1500 1000 1800 900
oxooxooxoxxxxxxxxoxxxxoxoxoxxooxoxxoxxxxxoxoxxooooooxooxooooxoxxoxooxooxxxoooooxxxooooxxxoxoxxooooxo
xoooxooxoxooxooxoooxxxoxxxooooxxoxxxxxooxxoooxoooooxoooxxxxooxxxoxooxxooooooooooxoxxxooxxxoxxoxoooox
xxoxooxxoxoxxxxxxxooxxooxoxxxxxxxxooxoxxoooxxoxxoxooxxxxoxoxxxxoxxooxoxooxoxxxoooxxxoxxoxoxooooxoxox
xxoooxxoxooooxxooxooxxoxxxxxoxxxooooooxoxxooxoxoooxxooxxxxxoxxxxoxxxoxxxxoooxxoxxoxoooxxooxooooxxoox
xoxoxxoooxoxxxooooxoooxoooxxoxooxxxoooxoxxxxooxxxoxxooxxxxxxooxxxoxxooooxooooooxxoxoxooxoooxxxxxoxxx
xoxooxoooxxxxxooxoooooxxooxoooxxoooxooxoxooxxoxxoooxxxooxooxxooxoxxxoxxoooxxxxoxxoxxxoxooooooxoxxxox
xxxoxxoxooxxooxooxxoooxxoooxooxxxoxxoxoxxoooxxooxoxxoooxxxxxoxxxoxooooxxxoxooxooxooxoxoxxoxxoxooxxxx
xxooooooxxoxooooooxxoxooxooxxoxooxxoooxxooxxxxxoxxxxxooxoooxooxoxoooxxxoooooxxxoooxxxxooxoxooooooxxx
ooxooxxxooxxxooxxxxxxxxooxxoxooxoxxooooxoooxoxoxxoxxoxoxxxoooxxoxoxooxooxooxxoxoxxxoxoxoxoxxxoxxxoox
xoxooxxooooooxoxoxxoooxoxxoxxoxxoxoooxooxxoxoooxxxoxxooxooooooooxooxxooxoxxoooxoxoxoxxxoxooooooxoxxx
oxxxoxxoxxxoxxooooxoxxxxoxxxoxxooxoxooxxxoxoooxxoxoooxxoooxoxoxxxoxoooxxoooxoxxxoooxoxxooxxxoxxxxxox
xxoxxxoxxxxoxoooooooxooxoxxoooxoxoxoooooxxxxxxxooooooooxxoxoxxoxooooooxxxxxxooxoooxoooxoxoxoxoxxxxxo
oxxxoxooxooooxxxoooxxoxxoxxoxxxoxxoooxxxxxxooxxxxooooooxxooxoxxoooxooxxxoxoxoooxoxxoooooxooooooxoxoo
oxxooxooxoooxoxxxooxoxoxooxooxxooxoooooxxoooxxxxooxxoxxoxxxxooooooxxooxxxxoxoooxooxoxxxxxoxooxxooxoo
xxoooxooooooxxooooxoooxxxxoooooooooooxoxooxoxxxoooxoxxxoxoooxoxxxxooxxooxoooxxoxxxxxxxoxxoxxxxxoooox
xxoxxooxxoxxooxooxxxoxoooxxxoxoxxxxoxxxoooooooxoxxooooxxxxooooxooxxxxoxoxxxxxxxxooxoxoooxooxoxxxoxxx
xxxoxxoxxooxxxxooxxoooxooxxxooxoooooxxoooxooxxxxxooxxxoxxxooxoxxxooxooxoxoxxxxxxxoxooooxoxooxxxxxoxx
xxooxxoxxoxoxxxoxoxoxxxoooxoxoooooooooooxxoxoxoxoxoxoooxxoxooooxxxxxxxxxooxxxoxxxoxooooxoxooxxxxxxxo
xxoxxxooxxxxoxoxooxoxoooooxoxoxxooxooxxxxxoooooooxxxooxxoxxxxxxoooooxxxoxoooxooxoxxxxxooxooxxxxoxooo
xxxxxxxxxoooxoxooxooxooxooxooxooxxxxoxooxooooxooooxoxoxoooxooxxoxoxxoooxooxxxoxoxxoxoooxoxxoxxxoxxox
ooxxoxooooxxoxooooooxooxoxooxooxxxoxoooxooxxooxxoxxoxooooxooxxoxxoooxxxooxoxxxxoooxxooooxooxxxxxooxx
ooxxooxxxooooxxxxoxoxxxxooxxxxxxoxoooooxoxxxxooxxoxxoxxxoxoooooxooxxxoxoxoxooooooxooxxooxxxoxoxxoxxx
ooxxxxooxooooxxxxoxoooxxoxxxxxooxoooxoxxoooooxoooxxxooooxxooxooooxxxxooxxooxxxxoxooxxoooooxxxooooxxx
xxxxoxxoxoooxoxxxxoxoxxoxoxxoxoxoxoxxooooxxooxooxxxxoxoxxxxxoooxoxxxxxxoxxoooxxoooxxoooxoxoxoooxooox
oooooxoxxxoooxoooxxoxoxoxxxooooxoooxooxoooxxxxxoxxoxxxxoooxxooxxoooxoooxoooxooxoxooxoxoxxooooxooxooo
ooooxoooooooxoooooxxoxxxoxxoxooxoxxoooxoxoooooooxooxoxoxoooooxxxoxxoooxoxxoxooooooxooooxxoxxxoooooxo
xxxxxxxoxoxxoxxooxoxxooxxxxxoxxoxoooooxoooxxxxxooxxxxxxooooxooxxoooxoxooxoxooxxxxxoxoxxxxxoxxxxoxxoo
ooxxooxxxxxxoxoxoxooooooooxxxxooooxoxxxooxxooxoxxxoxxxoxxxxooooxoooxoxxoxoxooooxoxooooooooooxxooxoox
xxxooxxoxoooxxxxxoxoooooxxoxooooxxxxxooxooxxoxxooooxoxooooooxooooxooxxoxxxxoxxxoooxxooxoooxxooxoxxoo
oxxxoxxooooxxooxooxxxoxooooxxxoooxoxoooxxoooxxooxxoooxoxooxxoxoooxoxoxoxxxxoxoooxxxxoxxxoxooooooxxxx
oxxxooxxoxooxxooxxxoooxxoooxxxxxooooxoxxoooooxoooxoxxxoxxoooxxxxxoooxxoxxoxooxooxxxxooxoxooxxxooooxx
ooooooooxxxooxoooooxooxoxoooxoxoooooooxooxooxxxoooooooxoxoxoooxoxooxxooooxoxoxxxxooxooxxxxooooxxoxox
oxxxxoxxxoxooooxxoooxooooooxoooxxooxxooxxoooxxoxoxooooxoooxxxxoooxxoxooooxoxxxxoxxxxoxoooxxooxoxooox
oxxoxoxooxoxoxxxxxxoxoxoooxooooxoxxxooxoooxxoxxxoxoxoxoxxxxxxoooxoxoxxxxoooxxxoooooxxoxoxoxxxooxooxo
oooxxxoxoxooxooxooooxxxoooooxooxxxoxxooooxoxoxxxoxxxxxxxoxxxoxxxooxxooooxxxooooxooxxxxoxooooxoxxooox
xxoxxoxxoooxoooxxoxoxxxxoxxoxooxxxxoxooxxxoxoxxoxoxoxxxoxoooooxxoxoooooxoooxxxxoxxooooooxooxxooxooxx
ooooxxxooxoooxooxooxxxooooxoooooxoxxxoxoxxoooxxoxooxxoxoxoooxxoxxoxxoooxoooooooooxxxxoxxooxoxoxoxoxx
xoxooxxxxoooxxxooxoxxxoooooooooxooxooxxoxoxoooooxxxxxxxxxxooxxoxxooxoxoxoooxxxxoooxoooxoxxxooxxoxxoo
ooxooooxxxxoxxooxxoooxxoxxoxxoooxxooooxxxoooxxoxoxooxoxxxxxooxxxooooxoxooooooxxoxooxoooooxoxooxxxxoo
xoooxoooxxoooxxxxxooxxxxxxxooxxoxxxxxxxoxxoxxoxxxoxxxxooxoxxoooxoxxoxxxxoooxoxxoxxoxoxxxxxxxoxoooxoo
xooxoooxoxxoxxooooxxoxoooxoxxxxxoxxoooooxxxxxxxxooooooxxoooooxoxoxxxxoxoooxxooxxooxooooxxoooxoxoxoox
xxxxxoxxxxxoxxooooxoooxooxoxooooooxooxxooxoooxxxoxxoxxooxxxxoooxxoooxxoxxxoooooxxxxxoxxooooxxxxoxoox
oxooxxxxooxoooooooooxooxoxxxoxxxxoxxxxoxxooxoxxoooooxxxxxooooxxxxooxoooxoxooxxxxooxooooooxxxoxxooxoo
xxxxxxxoxoxooxooxoxooxxoooooxoxxxoxxoxooooxoxooxxxooooooxoxxooooxxxooooxxooooxxooooxxooxoxooxxxxxxox
xoxoxxoxxoxxoxoxoooxooxooooooxoooxxooxxoooxxxoxooxxxoxxxxxoxoxxxoxooxxoxoxoooxxxoxxoxooxooooxxooxxxo
oxoxoooooooooooooxxxoxxoooxxxoxoxxxoooxooxoxxxxoxoxoxoxxooxoxooxooxxoxxxoxooxxxooxoooxooxoxoxoxoxxxo
xxxxooxxoooooxoxxxooxxxxxxxxoooxoxooxxxoxoooooxooooxxxxxxxoooxooxoxxxoxoxooooxoxxoxxoooxxoooxxoxooox
xxoxoxxooxoxooxooxxoxoxxxxoooxxoxxooxxxxxooxxxoxxxxoooooxxooooxooxxoooxxxooxxxxxooxxxxoooxxxoxxoxoxx
xxxoxooxooxooooooxxxoxooooxxxoxxxxxxooxoxxoxxoxoxxoxoxxxxxxxxxoxxooxooxxoxxxxxoxxooxoxoooooooxoxxoxo
xoxoxoooxxxoooxoooxxxxoxoxooxooxoooxoxxoooxxxxxooooooxoxoooxxxxooooxxooooxxoxxxxoxxxoxxoxooxxxxxxxxx
oxxooxxooxxoooxxoxxoxooxxoxoxxxoooooooxoooxxxooooxoooooooxxxxxxxxxooooooooxxxxoxooxxxooxoxxooooxooxo
xoxoooxxxxooxoxxooxoxoooooxxxoxooooooxxoxxxxooooxoxxooxxxxoxoxxxxxxooxoxxoxxxxoxooxxoooxooxxxxoxooxo
oooxoooxoxxxxxxoxxxxxxxooxoxooxxoooxxxxooooooxxxxxxoxooxxooxooxxxooxxoxooxoxoooxxoxoxxxoxooxoxxoxxox
oooxooooxxxooxxoxoxxoooxoooxxxoooxoxoooxoxxooxxoxxoxxooxoxoxoxoxxooxxooxoooxooooxxoxoooxoooooooooxxo
oxxxxxooooxoxooxxxooooxoxoxxooxoxooxxxxoooooxxoxooxoooxooxxxxoxoooxxoooooxxoxxooxoxoxooxoxxoooxxoooo
xooooxxxoooxoooxooxxxxxxxoxxoooxxxxoooxxxooooxxxxxxxxoxxooooooxooooxxxxooxoxxoxoxooxxoxooxxooxoxxoox
xxxxxoxxoxoooxoxxoxoooxoxxoxxxoxoooxxoxxxxxoooooxxxoxoxxoxxxooxoxoxoxoxxoooxooxxxxoxxooxxoxxoxooxoxx
xooxxoxoxxxoxooxoxxxooxxooxxoxxoxoooooxxxooxxxoooooooxoxoxxooxoxoxxxxxxoooxoooxxoxoooooxoooxooxooxoo
xxooxxxxoxxoxxxxooxxoxxoxxoxoxxxoxoxxxooooooxoxxoxxxoxoxxxxoxooxoxxooxooxxooxoxxxxxoooxoxoxoxoxxoxxx
oooxxooxxooooooxxooxxoxxoxoxxoxoooxxooxoxxooxxxoxoxxxoxxxxxxxxoxxxxxxxxoxxxooxooxxoooxoxxxxoxxooxooo
xoxoxooooxxxoxoxxxoooooxooxxooxooxooxxoooxxxoxxxxoxoooxooooxxxxxoooooxxooxxxxoxooxooooxxxxooxxooxxxx
ooooooxoxoxooxooxxoxxoxxoxooooxoxoxoxoxoxxooxoxoxxooxooxxooxoxxxoooxoooxoxoxooxooooxoxxoooxooxoooxox
xxooxxoxooxoxxoxoxooxxxxooxoxxoxoxooooxoxxoooooooxxxxooooxoxoxoxooxxxoxoxxoxxxooxoooxoxoxoxoxxoxoxxx
xoooxoxxooooxoxoxxooxxxooxxxoxoxoooxoxooxxoxooxxxxoxxooxxooxooxoxooooxoxooxxoooooxoooxxxoxoxxoxoooxx
ooxxooxoooooxxoooooxxooooooxxoxoooxooxxxxooxoxooxoxoxxxoxoooooxoooxxoooxxoxxxxoooxoooxooooxoxxxxxxoo
ooooxoxxoxoxooxxxxxooooxoxxooxoxxxxooxxxxxoxxxooooxooooxooxoooxooxoooxxxooooxooxxxoooooxxxxxoxoxxxxx
xoxxxxxxoxxxxxoooxoooxxxxxoooxoxxxxxxooxxooxxxooxooooooxooxooxxoxxxxxxxoxxxooxxxoxxxxxoxxoxxxxoxooxo
xxooxxoxooxxxoooxxoxxoxxoxxxoxoooxooxoxxxoooooxoxooxoxoxoxxooooooxoxxxoxxxxxoxoxxooxoxooxoooxxxoxxox
ooxxxoxxxoooxxoxoxxoxoxoxxoooooxoxxxooooooxxxoooxxooxoxxoxoooxxxoooooxxxoxooxoooxxoooxoxooxoxooxxxxx
oxoxxxooxxxoxoxooxoooxooxxxoxoxooxxoxxxoooxxxxxoxooxxxxxxxoxoooxooxoxxxooxoxxoxooxoxxxoxoooxxxoxxoxx
ooxoooxooxooooooxxoxoxoxooxooxxooxoxxxxxxxoxxxooooooxxoooxoxooxoxxooxoooxxxxxoooooxooooxoooxxoxxoxxx
oxxoxxooxxoxxxxxooxoxxxoxxxoooxooxxooxoxxoxxxooxxoxoooooooxoxxxooooxxoxxooxoxooxxxxxoxooxxoxxoooxooo
oxxoxxxxoxooxxoxoxxxxooxxxxoxoxoooxxxooxooxooooxooxxooxooxooxxooxxooxooooxxooxxooxooxooxooxxoxooxxox
oxxoxoxxoxooxoxxoooxxxxooxxoxxooxxooxoxxooooxxxoxoxooxxoxooxoxoxooxxooxoooxoxooooxxxooxoooxxoxooxxox
oxoxxoxooxxoxoxoxoxxxxoxooxoxxxooxxooxxxooxxoooxxooooxxooxoxxxoxxoxxxxooxoooooooxxoxxxooooxxxxxxxoxo
oooxoxooxxxxoxoxooxxxxoxxxxoxooxoxxoxoooxoooooxoxoxxoxoooooooxxoooooooooxxooxoxoxooxxxooooxooxoooooo
xxoooxxxooooooxooxooooxoxooxooxxoxoxoooooxxxxooxxxooxxoxxxxooxxxxoooooxxxoxxxoxoooxoxxooxxxoooxoxxox
ooooxxoooooxxxoooooxooxxooooxxxooxoxxooxooxxxoxoxooooxoxooooxoxooxooooxxxxoooxooxoxoooooxoxxxxxxooox
oxoooxoxoxoxoxooxxoxoooxoxxooxoxxoooooxxoxoxooooxooxxxooooxoxxoooooooxxoxxoooxoooxooxoxxooxxxooxoxox
oxxoxxoxxoxxoooxoxxxxxxxxoooxxoxooooooxxxoxooooxooxoxoxxxoxxxxxoxxooxxoooxxxxxooooxxooooxxxoxooxxooo
xoxxxoooxxxoxoooxxoooxoxooxxxxxooxoooxoxxxxoxoxxoooxxxxooxooxoxoxxoxoxxooooooooxxoooxxxoxoooxxoooxoo
oxoxooxoxxxxoxxxoxoxoxxoxooxxxoxxoxoxoxoxxxxxxoxoooxoooxxooxoxoooxxoxooooxxxoooxxxoxxxoxxxxxxooooxxo
xoxooooxoxooooxoxoxoooxooxoxoooooxoxooxxxxxxoxxoxxxxoxxooooxoxxoooxxxxxxooxxxxxooxxxooxxxxoxoxxxoxxx
xoxooxxxoxxxoxooxoxoxxxoxoxoxoxxxoxoxxoxxoxxxxoxxoxoxxooooxxxxoxxxxxxooxoxoooxxooxxxoxxoxoxooooxxxoo
xoxooxxooooxoxxxoxoooooooxxxoxxooxxxoxooxoxxooooxooooooooxxxxooxooxoxxxxxooxxooxoxooxoooxoxoooxxxoox
xoxxxxxxoooxooooooxoxoooxxxxoxoxooxxoxoxxxoxxxxoooxxxoooxoxoxooooxooooxooxoxoxxxxxooxxoxxoxoooxxooxo
xxxxoxoxxxoxxxoxxxoxxxxoxxxxxoxxooxoxxoooxxoxxooxxooxxoooxooxxxxooxxoxoooxxxooxoxxxooxxoxoooxxxxxoxx
xxxooooxooooxxooxooxxxooxxooxxoxxxxxxxooxxxooooxxoxooxxxxoooxooxxxoxxooxoooooxoxxxxxxooxxoxooooxooxo
oxxooxoxoxxxooooxxxoxxoxooxoxoxxoxxxxoxooxooxxxxoxoxxoooxoxxxooooxooxxoxooxoooxoooooooxooooxooxxxxxx
xooooxxxoxoxooxooxxooxxxxoxoxxxooxxoooooxooxxxooxoxoxxxoxxoxoooooxxooooxxxooxoxxoxoxxxoxooooxxoxxooo
xxooxxxooooooooxxoooooxoxooxoxxoxxxxoxxoooxoxooxoooxoxoxoxxxxxxxxxxooxoxoooooooxoxoxoxooxxxoooxoxoxx
xxxxxoxoxooxxooxooxooxooxoxxxoxxooxooxxoooxxoooxoxoxxxxooxxoxoxooxoxxxoxxxoxoxoooxxxxxxoxoooxoxxxoox
oxxoooxxxxoooooxxxooxooxxoxxxxoooxoxxoooxoxxxxxxoxooxooooxxoxxxxxxxoxoxoxxooxoooxxooxoxooooooxxooxoo
oxoooxxoxxxxxxxoxoxxoooxoxoxoxxxooxooxoxooxxxoxxoxoxoxxoxxxxxxoooxxoxxoxoxoxxxoxxxoxoxooooxxxoooxxoo
oxxxxoxoooxooxoooooxoooxxoxxxoooooooxooxoxxxxoxooxxoxooxxooxxoooxoooxxxoxxoxxooxoxoooxoxoooxooxxxoxx
oooxoxoooxxoxoxxxxxxxoxooxxxxxxooxxxoxoxxxooxooxooxxxoxoxxoxoxxoooooxxxoxxoxxooooxooxooxxoxxoooxxoox
ooxxooxxxxoxoxxoooxoxxxxxxoxxoxxxxoooxoxxxxxooxxxxxxoxoooxxxoxxoooxxooooxxxxoooooxoxoooxooooxxxooxxo
xxxoxxoxxoooxxxxoooxxoxooooooxooxxxxxxoxoxoxxxxoooxoxooxxoooxxxxoooooxooxxoxoxoxoxxoxoxooooooxoxxxxo
oxxoooxxxxoxxxxxooooxxxooxxxxoxxoxxooxooxxxxoxxoooxxoxxoooxoooxxooxooxxooxoxoxxoxoxooxoooxxoooxxooxo
xooooooxxxoxooxxxxoxxoxooxxooxxoooxoooxoxoxooxxooxoxxxxooooooooooooooooxoxooxoxooxxxoxooxooxooxoooxx
