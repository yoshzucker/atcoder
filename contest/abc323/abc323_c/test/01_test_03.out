13
9
17
16
14
12
18
8
15
6
17
10
11
11
11
15
15
14
15
11
8
13
10
16
7
0
20
8
8
11
13
2
8
14
11
12
8
12
9
21
7
15
10
11
14
8
12
19
17
14
9
14
16
6
8
13
17
9
20
16
10
7
12
9
5
13
22
14
9
15
6
14
10
11
15
5
10
4
5
13
11
16
15
19
7
12
21
12
12
11
11
14
10
19
7
14
14
14
14
5
